EIPS/eip-wrapped_deposit.md

```diff
@@ -0,0 +1,137 @@
+---
+eip: <to be assigned>
+title: Wrapped Deposits
+description: A singleton contract for managing asset deposits.
+author: Justice Hudson (@jchancehud)
+discussions-to: https://ethereum-magicians.org/t/wrapped-deposit-contract-eip/7740
+status: Draft
+type: Informational
+created: 2021-12-11
+---
+
+## Abstract
+The wrapped deposit contract handles deposits of assets (Ether, ERC20, ERC721) on behalf of a user. A user must only approve a spend limit once and then an asset may be deposited to any number of different applications that support deposits from the contract.
```

Comment:
```suggestion
The wrapped deposit contract handles deposits of assets (Ether, [ERC-20](./eip-20.md), [ERC-721](./eip-721.md)) on behalf of a user. A user must only approve a spend limit once and then an asset may be deposited to any number of different applications that support deposits from the contract.
```
(EIP-1 specifies hyphenation - I generally forget about this myself, as other editors here can testify 🙂)